hack/validate-imports/validate-imports.go

```diff
@@ -209,6 +214,7 @@ func validateImports(path string, fset *token.FileSet, f *ast.File) (errs []erro
 		"pkg/database/cosmosdb/zz_generated_",
 		"pkg/operator/apis",
 		"pkg/operator/clientset",
+		"pkg/operator/mocks/",
```

Comment:
I wasn't so sure about putting Operator controller specific mocks into the util directory, just in case the Operator ends up being pulled out/separated/etc as a part of future things. Similar to how the API is kept in the operator, not with the rest of them.